cartoframes/viz/helpers/animation_layer.py

```diff
@@ -26,16 +35,34 @@ def animation_layer(
         source,
         style={
             'point': {
-                'color': 'opacity({0}, 0.8)'.format(color or '#EE4D5A'),
-                'filter': 'animation(linear(${0}), 20, fade(1,1))'.format(value)
+                'width': '{0}'.format(
+                    size or defaults.STYLE['point']['width']),
+                'color': 'opacity({0}, {1})'.format(
+                    color or '#EE4D5A', opacity or '0.8'),
+                'strokeWidth': '{0}'.format(
+                    stroke_width or defaults.STYLE['point']['strokeWidth']),
+                'strokeColor': '{0}'.format(
+                    stroke_color or defaults.STYLE['point']['strokeColor']),
+                'filter': 'animation(linear(${0}), {1}, fade{2})'.format(
+                    value, duration or 20, fade or (1, 1))
             },
             'line': {
-                'color': 'opacity({0}, 0.8)'.format(color or '#4CC8A3'),
-                'filter': 'animation(linear(${0}), 20, fade(1,1))'.format(value)
+                'width': '{0}'.format(
+                    size or defaults.STYLE['line']['width']),
+                'color': 'opacity({0}, {1})'.format(
+                    color or '#4CC8A3', opacity or '0.8'),
+                'filter': 'animation(linear(${0}), {1}, fade{2})'.format(
+                    value, duration or 20, fade or (1, 1))
             },
             'polygon': {
-                'color': 'opacity({0}, 0.8)'.format(color or 'TODO'),
-                'filter': 'animation(linear(${0}), 20, fade(1,1))'.format(value)
+                'color': 'opacity({0}, {1})'.format(
+                    color or defaults.STYLE['polygon']['color'], opacity or '0.9'),
+                'strokeColor': '{0}'.format(
+                    stroke_color or defaults.STYLE['polygon']['strokeColor']),
+                'strokeWidth': '{0}'.format(
+                    stroke_width or defaults.STYLE['polygon']['strokeWidth']),
+                'filter': 'animation(linear(${0}), {1}, fade{2})'.format(
```

Comment:
Same as above! :) Just so we don't forget :)

Reply:
Yep! You were 100% right. Actually, it's a new parameter and now that I take I look, I'm gonna si if we want it as a string with or without parenthesis, I'll let you know 👍